### PR TITLE
ADM-361 [frontend] [Metrics] Crew Settings - add e2e test

### PR DESCRIPTION
**Pull request by fengxinhou**
## Summary

Add e2e test

### CODE REVIEW

File: frontend/cypress/pages/metrics.ts [hunk 1]
```diff
@@ -31,6 +27,17 @@ class Metrics {
...
 
+  verifyJiraBoard() {
+    cy.intercept(Cypress.env('url') + '/api/v1/boards/*', (req) => {
+      req.url = req.url.replace('/v1/', '/v2/')
+    }).as('verifyJira')
+
+    cy.contains('Verify').click()
+
+    cy.contains('Verified').should('exist')
+    cy.contains('Reset').should('exist')
```
Comment (guzhongren) [inline frontend/cypress/pages/metrics.ts:38]: should not contain any assertion in POM

File: frontend/cypress/pages/metrics.ts [hunk 2]
```diff
@@ -54,6 +61,17 @@ class Metrics {
   goMetricsStep() {
     cy.contains('Next').click()
   }
+
+  selectCrewSetting() {
+    cy.wait('@verifyJira').then((currentSubject) => {
+      cy.contains(`${currentSubject.response.body.users.join(', ')}`).should('exist')
```
Comment (guzhongren) [inline frontend/cypress/pages/metrics.ts:67]: Same as the above

File: frontend/cypress/pages/metrics.ts [hunk 3]
```diff
@@ -54,6 +61,17 @@ class Metrics {
...
+    })
+
+    cy.contains('Included Crews').siblings().click()
+
+    cy.get("[type='checkbox']").should('be.checked')
```
Comment (guzhongren) [inline frontend/cypress/pages/metrics.ts:72]: Same as the above

File: frontend/cypress/pages/metrics.ts [hunk 4]
```diff
@@ -54,6 +61,17 @@ class Metrics {
...
+    cy.get('div.MuiBackdrop-root.MuiBackdrop-invisible.MuiModal-backdrop').click({ force: true })
```
Comment (guzhongren) [inline frontend/cypress/pages/metrics.ts:73]: Can we use another method to get the element, 
classname is not stable for e2e testing when upgrading the 3rd package

**Review by guzhongren — COMMENTED** [review]
Change with the comments

**Review by tingwang1234 — APPROVED** [review]

**Review by LanxinWang — APPROVED** [review]
LGTM

**Pull request merged**